index.html

```diff
@@ -23,47 +23,66 @@
             <div class="inner">
                 <h1>Gci16.fossasia.org</h1>
                 <h2>GCI Website 2016</h2>
-                <a href="https://github.com/fossasia/gci16.fossasia.org" class="button"><small>View project on</small> GitHub</a>
+                <a href="//github.com/fossasia/gci16.fossasia.org" class="button">
+                    <small>View project on</small> GitHub</a>
             </div>
         </header>
 
         <section class="main-content">
             <div class="container">
-            		<section id="contributors">
-			            <h2>CONTRIBUTORS</h2>
-			            <div id="contributors-list">
-            			</div>
-        				</section>
+                <section id="contributors">
+                    <h2>CONTRIBUTORS</h2>
+                    <div id="contributors-list">
+                    </div>
+                </section>
                 <div class="students-section">
                     <h1 class="head"><span>Students</span></h1>
                     <hr>
                     <div class="row">
-                        <div class="col-lg-3 col-md-3 col-sm-4">
+                        <div class="col-sm-4 col-md-3">
```

Comment:
Redundant, `col-md-3` exists. Since it doesn't change the column size, it's not required.
Here's the linter message for it.
```
index.html
|  41| ••••••••••••••••••••••••<div•class="col-lg-3•col-md-3•col-sm-4">
|    | [MAJOR] BootLintBear:
|    | Since grid classes apply to devices with screen widths greater than or equal to the breakpoint sizes (unless overridden by grid classes targeting larger screens), `class="col-lg-3 col-md-3 col-sm-4"` is redundant and can be simplified to `class="col-sm-4 col-md-3"`
```